src/middleware/fhir/fhir-response-util.js

```diff
@@ -0,0 +1,149 @@
+const path = require('path');
```

Comment:
Copy and change this code from the node-fhir-server-core so it can support streaming